base/commandLine.cpp

```diff
@@ -305,7 +305,7 @@ void registerDefaults() {
 	ConfMan.registerDefault("tempo", 0);
 #endif
 #if (defined(ENABLE_SCUMM) && defined(ENABLE_SCUMM_7_8)) || defined(ENABLE_GRIM)
-	ConfMan.registerDefault("dimuse_tempo", 10);
+	ConfMan.registerDefault("dimuse_tempo", 50);
```

Comment:
now I thinking, better to remove this option. old settings may have this which will not be good for new implementation. it was for slow disk access tuning

Reply:
If I remember correctly, I wrote some code which checked the number and updated it to the new rate. But yes I agree it would be better to remove it. I will do that